### PR TITLE
REAME.md, CONTRIBUTORS: update

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
REAME.md, CONTRIBUTORS: update

@jegj @benformosa @rgl @Boosai @gotmax23 @jmighion @SantiRaposo @Vanav @Jhiliano @btoussaint @amossc @codrinh thank you for contributing to this project!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,9 +73,9 @@ The PostgreSQL modules rely on the [Psycopg2](https://www.psycopg.org/docs/) Pos
 ## Tested with ansible-core
 
 Tested with the following `ansible-core` releases:
-- 2.11
 - 2.12
 - 2.13
+- 2.14
 - current development version
 
 Ansible-core versions before 2.11.0 are not supported, in particular, ansible-base 2.10 and Ansible 2.9.
```
Comment (hunleyd) [inline README.md:81]:
```suggestion
Ansible-core versions before 2.12.0 are not supported, in particular, ansible-base 2.10 and Ansible 2.9.
```

File: CONTRIBUTORS [hunk 2]
```diff
@@ -205,6 +216,7 @@ tomscytale
 Trikke76
 truki
 tYYGH
+Vanav
 veger
 vfoucault
 vmalloc
```
Comment (hunleyd) [inline CONTRIBUTORS:222]: should we maybe use something like [this](https://github.com/marketplace/actions/auto-add-contributors) to auto-maintain this?

**Review by hunleyd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Andersson007) [reply README.md:81]: @hunleyd maybe we could remove what follows after 2.12.0? If you agree, feel free to update your suggestion and I'll commit it

File: README.md [hunk 3]
```diff
@@ -47,6 +47,13 @@ We announce important development changes and releases through Ansible's [The Bu
 
 Join us in the `#postgresql:ansible.com` [room](https://matrix.to/#/#postgresql:ansible.com) on Matrix, the `#ansible` (general use questions and support), `#ansible-community` (community and collection development questions), and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels) on [Libera.Chat](https://libera.chat).
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md:48]: Not sure i fully understand, feel free to suggest in code what you'd like to improve

File: CONTRIBUTORS (hunk 2 above)
Comment (Andersson007) [reply CONTRIBUTORS:222]: i also add issue reporters and folks who take part in discussions in comments. Does the tool support it? Also I see security tokens mentioned, would it be safe to expose it?
Also I like doing it periodically. It reminds contributors that we're happy to see their contributions after some time passed.

Sorry for the noise folks, I should've thanked you after this PR gets merged

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (hunleyd) [reply README.md:48]: I'm saying I think line 48 can be removed, given the new content. And then I attempted to say that the new content you added doesn't mention the postgresql room on Matrix

File: README.md (hunk 1 above)
Comment (hunleyd) [reply README.md:81]: like that?

File: CONTRIBUTORS (hunk 2 above)
Comment (hunleyd) [reply CONTRIBUTORS:222]: ah, no it doesn't cover it all. i withdraw the suggestion. carry on :)

File: README.md (hunk 1 above)
Comment (Andersson007) [reply README.md:81]: exactly, thanks!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md:48]: It's there as i can see, the first item in the list, in line 52, isn't it?

[the remark this answers is not in the document]
Comment (jchancojr) [reply README.md:48]: Line 48 seems to reiterate lines 50-59 maybe?

[the remark this answers is not in the document]
Comment (hunleyd) [reply README.md:48]: the text currently reads as:
```
Join us in the #postgresql:ansible.com room on Matrix, the #ansible (general use questions and support), #ansible-community (community and collection development questions), and other IRC channels on Libera.Chat.

Join us on Matrix in:

    #postgresql:ansible.com room: questions on how to contribute and use this collection.
    #users:ansible.com room: general use questions and support.
    #ansible-community:ansible.com room: community and collection development questions.
    other Matrix rooms or corresponding bridged Libera.Chat channels. See the Ansible Communication Guide for details.
```
The first para is redundant w/ the bullet list, no?

[the remark this answers is not in the document]
Comment (hunleyd) [reply README.md:48]: so, i suggest:
```
Join us on Matrix in:

    #postgresql:ansible.com room: questions on how to contribute and use this collection.
    #users:ansible.com room: general use questions and support.
    #ansible-community:ansible.com room: community and collection development questions.
    other Matrix rooms or corresponding bridged Libera.Chat channels. See the Ansible Communication Guide for details.
```

[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md]: Ah, i though i'd removed that... :D sorry, done, thanks

**Review by jchancojr — APPROVED** [review]
lgtm

**Comment by Andersson007** [comment]
@hunleyd @jchancojr thanks for reviewing!

**Pull request merged**